### ISSUE TITLE
Empty view message: links should have some kind of aria-label

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#64062]
It looks like the links in empty view messages do not have any form of aria-label that would assist a user that is using a screen reader from explaining what the link does. It should probably read something like "Open Reference: the reference..." e.g. for the references view.

**Comment by bpasero** [comment]
Looks like this is already possible with providing a title to the links, closing.
